tests/test_smoke.py

```diff
@@ -968,6 +968,22 @@ def test_new_bucket_creation_and_deletion(self, tmp_local_storage_obj,
         out = subprocess.check_output(['sky', 'storage', 'ls'])
         assert tmp_local_storage_obj.name not in out.decode('utf-8')
 
+    def test_sky_storage_cli(self, tmp_bucket_name, tmp_source):
+        # Creates new bucket with local source
+        out1 = subprocess.check_output(
+            ['sky', 'storage', 'create', tmp_bucket_name, tmp_source, 'AWS'])
```

Comment:
`AWS` should be `S3`. Might want to double check if the test passes.

Reply:
Changed to S3 in test and corresponding methods.